Gemfile

```diff
@@ -31,6 +31,7 @@ gem 'gettext_i18n_rails', '~> 1.0'
 gem 'rails-i18n', '~> 5.0.0'
 gem 'turbolinks', '>= 2.5.4', '< 3'
 gem 'logging', '>= 1.8.0', '< 3.0.0'
+gem 'logging-journald', '~> 1.0'
```

Comment:
We could pre-emptively open a packaging PR for this.

Reply:
Working on it right now.

Reply:
I am extracting this into `bundler.d/journald.rb` so developers can easily do `bundle install --without journald` in order to avoid installing `systemd-devel`.